orca-clouddriver/src/test/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/manifest/ManifestForceCacheRefreshTaskSpec.groovy

```diff
@@ -0,0 +1,43 @@
+package com.netflix.spinnaker.orca.clouddriver.tasks.manifest
```

Comment:
Missing license header, see other files as an example. Should include `Datadog`, check w/ your legal for details.